docs/source/installation/guide_quick/index.rst

```diff
@@ -0,0 +1,83 @@
+    Please refer to the `Python Package Index <https://pypi.org/project/aiida-core/>`_ for the minimum required version.
+
+First, install the ``aiida-core`` Python package:
+
+.. code-block:: console
+
+    pip install aiida-core
+
+Next, set up a profile where all data is stored:
+
+.. code-block:: console
+
+    verdi presto
+
+Verify that the installation was successful:
+
+.. code-block:: console
+
+    verdi status
+
+.. code-block:: console
+ ✔ version:     AiiDA v2.5.1.post0
+ ✔ config:      /path/.aiida
+ ✔ profile:     presto
+ ✔ storage: SqliteDosStorage[/path/.aiida/repository/sqlite_dos_8ee66fde9bd34e7c8fc8e0e29b20c429]: open,
+ ⏺ broker:      No broker defined for this profile: certain functionality not available.
+ ⏺ daemon:      No broker defined for this profile: daemon is not available.
+If all lines show green checkmarks, the installation was successful and you are good to go.
+
+.. admonition:: What next?
+    :class: hint
+
+    If you are a new user, we recommend to start with the :ref:`basic tutorial <tutorial:basic>`.
+    Alternatively, check out the :ref:`next steps guide <tutorial:next-steps>`.
+
+.. warning::
+
+    Not all AiiDA functionality is supported by the quick installation.
+    Please refer to the :ref:`section below <installation:guide-quick:limitations>` for more information.
+
+
+.. _installation:guide-quick:limitations:
+
+Quick install limitations
+=========================
+
+Functionality
+-------------
+
+Part of AiiDA's functionality requires a `message broker <https://en.wikipedia.org/wiki/Message_broker>`_, with the default implementation using `RabbitMQ <https://www.rabbitmq.com/>`_.
+The message broker is used to allow communication with the :ref:`daemon <topics:daemon>`.
+Since RabbitMQ is a separate service and is not always trivial to install, the quick installation guide sets up a profile that does not require it.
+As a result, the daemon cannot be started and processes cannot be submitted to it but can only be run locally.
+
+.. note::
+    The ``verdi presto`` command automatically checks if RabbitMQ is running on the localhost.
+    If it can successfully connect, it configures the profile with the message broker and therefore the daemon functionality will be available.
+
+
+Performance
+-----------
+
+The quick installation guide creates a profile that uses `SQLite <https://www.sqlite.org/>`_ for the database.
+Since SQLite does not require running a service, it is easy to install and use on essentially any system.
+However, for certain use cases it is not going to be the most performant solution.
+AiiDA also supports `PostgreSQL <https://www.postgresql.org/>`_ which is often going to be more performant compared to SQLite.
+
+.. note::
+    The ``verdi presto`` command can also create a profile using PostgreSQL instead of SQLite.
+    Specify the ``--use-postgres`` flag and provide the connection information to the PostgreSQL service using the various ``--postgres`` options.
```

Comment:
I would mention them just to be clear
```suggestion
    Specify the ``--use-postgres`` flag and provide the connection information to the PostgreSQL service using the various ``--postgres`` options: e.g. `--postgres-hostname`, `--postgres-port`, `--postgres-username` and `--postgres-password`.
```

Reply:
I remember @mbercx objections to having these options for `verdi presto` in the first place, which I also agree with. Ideally, we just have `--use-postgres` here, without all the other `--postgres-...` stuff. I think the default db name (which is also currently a missing option `--postgres-name`) and db user seem are sensible, and if people want to configure more things, they should use `verdi profile setup`. Don't want to be too much of a scope creep here, but we should probably agree on this, as, once we add options and release, we have to keep them due to backwards compatibility...

Reply:
Couple of things here:

> I remember @mbercx objections to having these options for verdi presto in the first place, which I also agree with. 

I think you are thinking of his objection to having the RabbitMQ connection configuration options in `verdi profile setup`. This is addressed in #6480 .

What he _did_ remark with regards to the `--postgres-*` options is that he thought it wasn't consistent with the same options for `verdi profile setup core.psql_dos` where they are named `--database-*`. The problem is that we cannot change these for the `core.psql_dos` as they are taken from the pydantic model fields and are already part in the actual `config.json`. If we rename them, all configs need to be migrated as well. Alternative is to rename the options for `verdi presto` to use `--database-*` instead, but I find that less ideal because `--postgres-` is clearer.

> I think the default db name (which is also currently a missing option --postgres-name)

This is intentional. If you want the functionality that automatically creates a PSQL database, then why provide an option that let's you specify the name. In what use case would this be necessary?

> and if people want to configure more things, they should use verdi profile setup

But there they won't have the functionality that the database is created automatically, as you yourself would also want to keep. The options I added to `verdi presto` _are necessary_ to keep the automatic database creation functionality useful, as the connection details to the PSQL server are not always `postgres:postgres@localhost:5432` but the user, hostname and port can change. If the user cannot specify this, the command is useless.

Reply:
> I think you are thinking of his objection to having the RabbitMQ connection configuration options in verdi profile setup. This is addressed in https://github.com/aiidateam/aiida-core/pull/6480.

No, I talked with him the other day, and it was about the `--postgres-*` options of `verdi presto`. Though, let's maybe wait until he responds, as I don't want to put (even more ^^) words in his mouth.

Though, I also think that the fact that these are always shown is suboptimal, as they are not relevant for anybody using SQLite and might just cause confusion for new users. And I understand it's not possible to generate them dynamically like it is done for `verdi profile setup core.psql_dos`, which would be the ideal solution I suppose.

> What he did remark with regards to the --postgres-* options is that he thought it wasn't consistent with the same options for verdi profile setup core.psql_dos where they are named --database-*.

Yes, I also noted this in [my other PR](https://github.com/aiidateam/aiida-core/pull/6481#issuecomment-2178383526).

> The problem is that we cannot change these for the core.psql_dos as they are taken from the pydantic model fields and are already part in the actual config.json. If we rename them, all configs need to be migrated as well.

That makes sense, though, thanks for the clarification!

> This is intentional. If you want the functionality that automatically creates a PSQL database, then why provide an option that let's you specify the name. In what use case would this be necessary?

My reasoning is that, if one can specify all the other options for the db setup, then why not be able to also specify the name of the created db? I don't see the difference to the other options here, am I missing something?

> But there they won't have the functionality that the database is created automatically, as you yourself would also want to keep.

Fair enough, I didn't consider that there is no option to generate it automatically for `verdi profile setup core.psql_dos`. That being said, why not add an option there which enables this? As we ask for all these options anyway, why not give the user the choice to let it be created by AiiDA on-the-fly?

> The options I added to verdi presto are necessary to keep the automatic database creation functionality useful, as the connection details to the PSQL server are not always postgres:postgres@localhost:5432 but the user, hostname and port can change. If the user cannot specify this, the command is useless.

But is the command really "not useful" without these options? I guess for 99% of the intended users of the command, new users, they will have this setup, and will probably not even be aware of the existence of database user, hostname, and port options. Personally, I never found myself in a situation where I had to change these settings (though, my scope might be too narrow here). Assuming we have a flag for automatic database creation for `verdi profile setup core.psql_dos` as I propose above, removing the options from `verdi presto` and just running with the defaults seems sensible.

Reply:
> My reasoning is that, if one can specify all the other options for the db setup, then why not be able to also specify the name of the created db? I don't see the difference to the other options here, am I missing something?

The whole point of this functionality in `verdi presto` is that it can automatically create the required database (and user) in a PostgreSQL server. Typically, the user doesn't care about the details and so we just generate one. However, the user _does_ need to be able to specify where this server is. Although in many cases it may be running on the localhost on port 5432, that is not always the case. The same goes for the username and password with which to connect to PSQL. Although in some cases the defaults may work, or by resorting to using sudo, in some cases the user needs to specify a specific user account which has the rights to create a new database (and optionally a user).

> Fair enough, I didn't consider that there is no option to generate it automatically for verdi profile setup core.psql_dos. That being said, why not add an option there which enables this? As we ask for all these options anyway, why not give the user the choice to let it be created by AiiDA on-the-fly?

Because the commands are generated on-the-fly for each storage plugin and although the storage plugins can customize which _options_ are added, they cannot provide custom logic as would be required for the auto-resource creation for PSQL.

> But is the command really "not useful" without these options? I guess for 99% of the intended users of the command, new users, they will have this setup, and will probably not even be aware of the existence of database user, hostname, and port options. Personally, I never found myself in a situation where I had to change these settings (though, my scope might be too narrow here). Assuming we have a flag for automatic database creation for verdi profile setup core.psql_dos as I propose above, removing the options from verdi presto and just running with the defaults seems sensible.

I guess this is the crux of the disagreement/misunderstanding: _if_ we were to add the auto-generation functionality to `verdi profile core.psql_dos` then yes I would agree that it doesn't make sense to add it to `verdi presto`. However, this is currently not the case, and that is the reason I added it to `verdi presto` (which essentially replaces `verdi quicksetup`).

Reply:
> I guess this is the crux of the disagreement/misunderstanding: _if_ we were able to add the auto-generation functionality to `verdi profile core.psql_dos` then yes I would agree that it doesn't make sense to add it to `verdi presto`.

Yeah, I agree that's the issue... if there's no other way to have the automatic creation, then it should be at least under `verdi presto`.

> This is intentional. If you want the functionality that automatically creates a PSQL database, then why provide an option that let's you specify the name. In what use case would this be necessary?

I still don't get this, though... then why allow providing a custom `postgres-username`? The use case would be that maybe I'm too lazy to generate a PSQL database manually beforehand (e.g. because I don't remember the sequence of commands to do that), so I just let `verdi presto` do it for me, while I still have a specific db name in mind that I would want it to have?

Reply:
> while I still have a specific db name in mind that I would want it to have? 

Why though? The idea from the beginning was to keep `verdi presto` as automatic and simple as possible, only adding what was necessary. Initially that meant there was no automatic database creation. But you and other users still used that functionality from `verdi quicksetup`, which I agree makes sense. So I added that functionality, but only the _bare minimum_. I really think this is important. Because why when we allow specifying the database name, would we also not allow specifying the database username and password? This is unnecessary complexity that detracts from `verdi presto`.